receiver/prometheusreceiver/internal/otlp_metricfamily.go

```diff
@@ -170,10 +178,23 @@ func (mg *metricGroup) toDistributionPoint(orderedLabelKeys []string, dest *pmet
 	}
 	point.SetTimestamp(tsNanos)
 	populateAttributes(orderedLabelKeys, mg.ls, point.Attributes())
-
+	mg.setExemplars(exemplarsdata, point.Exemplars())
 	return true
 }
 
+func (mg *metricGroup) setExemplars(exemplarMap map[string][]exemplarData, exemplars pmetric.ExemplarSlice) {
+	metricKey := buildMetricKey(mg.ls)
+	ed := exemplarMap[metricKey]
+	exemplars.EnsureCapacity(len(ed))
+	for _, e := range ed {
```

Comment:
We need to deduplicate exemplars here.  See https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#histogram.  The only requirements are that each bucket uses an exemplar that is less than its bucket boundary, and an exemplar should be put into the highest bucket it can be placed in.  In the case of a single, small exemplar being part of a histogram, it will be placed in _all_ buckets.

Reply:
@dashpole Updated the code, Can you please check when you get time?